File: Day6/palindromeNumber.js
function getPalindrome(num){
    let temp = num;
    let reverse = 0;

    while (num != 0){
        reverse = (reverse * 10) + (num % 10);
        num = Math.floor(num / 10);  // use Math.floor() for integer division
    }
    return reverse==temp;
}
number = parseInt(prompt("Enter the Number:"));  // convert input to integer

let ans = getPalindrome(number);

if (ans)
    console.log(number + " is a palindrome number");
else
    console.log(number +  " is not  palindrome number");

File: Day6/palindrome_number.py
""" Program to check number is palindrome number or not."""

def get_check_palindrome(num):

    """ we store the original value in temp variable and then compare with reverse number"""

    temp = num
    reverse = 0

    while num != 0 :
        reverse = (reverse * 10) + (num % 10)
        num //= 10

    return reverse==temp

number = int(input("Enter the Number:"))
ans = get_check_palindrome(number)

if ans:
    print(number , " is a palindrome number")
else:
    print(number , " is not a palindrome number")
